src/Motions/Word.ts

```diff
@@ -124,11 +124,45 @@ export class MotionWord extends Motion {
 
         let targetText = document.lineAt(toLine).text;
 
-        toCharacter += MotionWord.characterDelta(
+        toCharacter += this.createCharacterDelta(
             targetText, this.wordDelta, toCharacter, option.isInclusive
         );
 
         return new Position(toLine, toCharacter);
     }
 
 }
+
+export class MotionWholeWord extends MotionWord {
+    protected wordSeparators = ' \t\r\n\f';
```

Comment:
Things like `\t` will be converted to real "tab" and then passed to regex builder.
We should escape `\` with `\\` so this will be ' \\t\\r\\n\\f`.

Besides, I think `\\s` should work too.